docs/content/latest/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Create a multi-zone universe using Yugabyte Platform and Kubernetes
+headerTitle: Create a multi-zone universe with Kubernetes
+linkTitle: Multi-zone universe
+description: Use Yugabyte Platform and Kubernetes to create a YugabyteDB universe that spans multiple availability zones.
+menu:
+  latest:
+    identifier: create-multi-zone-universe-kubernetes
+    parent: create-deployments
+    weight: 20
+isTocNested: true
+showAsideToc: true
+---
+
+  <ul class="nav nav-tabs-alt nav-tabs-yb">
+
+<li>
+    <a href="/latest/yugabyte-platform/create-deployments/create-universe-multi-zone" class="nav-link">
+      <i class="fas fa-building" aria-hidden="true"></i>
+Generic</a>
+  </li>
+
+  <li>
+    <a href="/latest/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes" class="nav-link active">
+      <i class="fas fa-cubes" aria-hidden="true"></i>
+      Kubernetes
+    </a>
+  </li>
+
+</ul>
+
+This section describes how to create a YugabyteDB universe in one geographic region across multiple availability zones using Kubernetes as a cloud provider. 
+
+## Prerequisites
+
+Before you start creating a universe, ensure that you performed steps described in [Configure the Kubernetes cloud provider](https://docs.yugabyte.com/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes/). The following illustration shows the **Managed Kubernetes Service** configs page that you should be able to see if you use Yugabyte Platform to navigate to **Configs > Cloud Provider Configuration > Infrastructure > Managed Kubernetes Service**:
+
+![img](/images/yb-platform/kubernetes-config1.png)
+
+Note that the cloud provider example used in this document has a cluster-level admin access.
+
+## Create a universe
+
+If no universes have been created yet, the Yugabyte Platform Dashboard looks similar to the following:
+
+![img](/images/yb-platform/kubernetes-config2.png)
+
+To start, click **Create Universe** and complete the first two fields of the **Cloud Configuration** section shown in the following illustration:
+
+![img](/images/yb-platform/kubernetes-config3.png)
+
+- In the **Name** field, enter the name for the YugabyteDB universe (or example, yb-k8s).
+
+- Use the **Provider** field to select the appropriate Kubernetes cloud (or example, K8s Provider). Notice that additional fields are now displayed, as shown in the following illustration:
+
+   ![img](/images/yb-platform/kubernetes-config4.png)
```

Comment:
Just an image as per the latest version.

![image](https://user-images.githubusercontent.com/13635843/123228207-ebe8aa80-d4f2-11eb-934c-286267cb4f62.png)

Reply:
Replaced image